test/unit_tests/aievec_tests/bf16_sqrt_2/bf16_sqrt.mlir

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// Copyright (C) 2023, Advanced Micro Devices, Inc.
+
+// REQUIRES: valid_xchess_license
+// RUN: aie-opt %s -affine-super-vectorize="virtual-vector-size=32" --convert-vector-to-aievec="aie-target=aieml" -lower-affine | aie-translate -aieml=true --aievec-to-cpp -o dut.cc
```

Comment:
A minor comment. Perhaps we can rename the test from `bf16_sqrt_2`to something like `bf16_sqrt_v32`? And the previous one to `bf16_sqrt_v16`, so that people can easily tell the difference between two tests.